.github/scripts/trymerge.py

```diff
@@ -123,6 +125,7 @@ def __init__(self, name: str, url: str, status: Optional[str]):
         workflow {
           name
         }
+        databaseId
```

Comment:
Honestly I'm not sure if the workflow id is needed, I'm pretty sure the job id uniquely identifies each job but w/e

Reply:
Ah, you're right, let's just compare the unique job id.  This will keep the change much simpler